content-watcher.js

```diff
@@ -93,10 +116,18 @@ const pollDirectoryForChanges = dirPath => {
 
 
 const pollerFunction = () => {
+  // Set the timeout first, to ensure an exception doesn't stop the recursion
+  setTimeout(pollerFunction, 1000)
```

Comment:
This should probably have been a separate commit - it seems useful on its own.

Reply:
Yeah, that's true.